src/FlashMessage/FlashMessage.tsx

```diff
@@ -154,7 +139,7 @@ const getBackgroundColorFromVariant = (variant: FlashMessageVariant) => {
             return colors.orange;
 
         default:
-            return '' as never;
+            return 'unset';
```

Comment:
This would then indicate that a variant not explicitly handled in the switch statement is Ok, I would rather indicate that the default case should never happen.

```suggestion
            return 'unset' as never;
```